### ISSUE TITLE
Gaussian (normal distribution) activation function

### ISSUE THREAD

**Issue by damiano1996** [tensorflow/addons#915]
**Describe the feature and the current behavior/state.**
After this [article](https://science.sciencemag.org/content/367/6473/83) I tried to implement a perceptron with an apical activation function similar to the real.
An approximation function of the apical was the gaussian, thus I implemented it.
In my [fork](https://github.com/damiano9669/addons) you can find it.

Now, also the perceptron can learn the XOR logic function:
![XOR](https://user-images.githubusercontent.com/41470459/72803694-fdb78580-3c4e-11ea-9198-397043809e37.png)

**Relevant information**
- Are you willing to contribute it? (yes/no): yes.
- Are you willing to maintain it going forward? (yes/no): no.
- Is there a relevant academic paper? (if so, where): I don't know.
- Is there already an implementation in another framework? (if so, where): I did not find anything.
- Was it part of tf.contrib? (if so, where): no.

**Which API type would this fall under (layer, metric, optimizer, etc.)**
activations.

**Who will benefit with this feature?**
researchers, developers.

**Any other info.**

**Comment by damiano9669** [comment]
What can I do? The code is just ready in my fork.

**Comment by seanpmorgan** [comment]
@damiano9669 Apologies for the delay in response. So I think this is an interesting idea, but I do not have a lot of intuition on how useful this is to the community at large. Without an academic paper describing the results and uses it's difficult to asses.

I would propose we leave this issue open and if community interests builds we could accept a PR for this. Alternatively if you proposed (and then included) a nice tutorial that shows its use and how it accomplishes something improved or novel then it would be a potential addition IMO.

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
